src/devices/Mhz19b/Mhz19b.cs

```diff
@@ -0,0 +1,244 @@
+        public Mhz19b(Stream stream, bool shouldDispose)
+        {
+            _serialPortStream = stream ?? throw new ArgumentNullException(nameof(stream));
+            _shouldDispose = shouldDispose;
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Mhz19b"/> class and creates a new serial port stream.
+        /// </summary>
+        /// <param name="uartDevice">Path to the UART device / serial port, e.g. /dev/serial0</param>
+        /// <exception cref="System.ArgumentException">uartDevice is null or empty</exception>
+        public Mhz19b(string uartDevice)
+        {
+            if (string.IsNullOrEmpty(uartDevice))
+            {
+                throw new ArgumentException(nameof(uartDevice));
+            }
+
+            // create serial port using the setting acc. to datasheet, pg. 7, sec. general settings
+            _serialPort = new SerialPort(uartDevice, 9600, Parity.None, 8, StopBits.One)
+            {
+                Encoding = Encoding.ASCII,
+                ReadTimeout = 1000,
+                WriteTimeout = 1000
+            };
+
+            _serialPort.Open();
+            _serialPortStream = _serialPort.BaseStream;
+            _shouldDispose = true;
+        }
+
+        /// <summary>
+        /// Gets the current CO2 concentration from the sensor.
+        /// </summary>
+        /// <returns>CO2 volume concentration</returns>
+        /// <exception cref="IOException">Communication with sensor failed</exception>
+        /// <exception cref="TimeoutException">A timeout occurred while communicating with the sensor</exception>
+        public VolumeConcentration GetCo2Reading()
+        {
+            // send read command request
+            var request = CreateRequest(Command.ReadCo2Concentration);
+            request[(int)MessageFormat.Checksum] = Checksum(request);
+            _serialPortStream.Write(request, 0, request.Length);
+
+            // read complete response (9 bytes expected)
+            byte[] response = new byte[MessageBytes];
+
+            long endTicks = DateTime.Now.AddMilliseconds(250).Ticks;
+            int bytesRead = 0;
+            while (DateTime.Now.Ticks < endTicks && bytesRead < MessageBytes)
+            {
+                bytesRead += _serialPortStream.Read(response, bytesRead, response.Length - bytesRead);
+                Thread.Sleep(1);
+            }
+
+            if (bytesRead < MessageBytes)
+            {
+                throw new TimeoutException($"Communication with sensor failed.");
+            }
+
+            // check response and return calculated concentration if valid
+            if (response[(int)MessageFormat.Checksum] == Checksum(response))
+            {
+                return VolumeConcentration.FromPartsPerMillion((int)response[(int)MessageFormat.DataHighResponse] * 256 + (int)response[(int)MessageFormat.DataLowResponse]);
```

Comment:
you should use BinaryConvert for the (DataHigh * 256 + DalaLow)

Reply:
@Ellerbach Laurent, do you mean BitConverter?
If so, I'm not sure if I feel comfortable with it. The BitConverter assumes that the byte-order of the array is equal to the byte order of the executing CPU architecture (same Endianness). Is this a valid assumption considering the multi-platform nature of .net Core? Of course I could check the endianness of the actual platform and reverse the array if required, but this seems to me a solution beyond the problem.

Reply:
ups, I meant BinaryPrimitives: https://docs.microsoft.com/en-us/dotnet/api/system.buffers.binary.binaryprimitives?view=netcore-3.1
Now in this case as MessafeFormat is a parameter, that may not be super straight forward. So I4m ok with that

Reply:
@Ellerbach Laurent, yes, BinaryPrimitives could make sense. However in this case I don't think that it would better compared to my current solution. Actually my implementation is even "closer" to the datasheet, if someone should look into the code and be not familar with endianness. 
So, we agree to keep it as is, I think.
I'll push the doc change.